scripts/tuple_tree_generator/tuple_tree_generator/templates/struct.h.tpl

```diff
@@ -26,24 +26,33 @@ struct /*= struct | fullname =*/
   /**- if struct.inherits **/
   static constexpr const /*= struct.inherits.name =*/Kind::Values AssociatedKind = /*= struct.inherits.name =*/Kind::/*= struct.name =*/;
   /**- endif **/
+  static const constexpr bool MODIFY_THIS_ONE = true;
```

Comment:
?

Reply:
Please don't silently mark as solved issues. I still see this.
If you mark solved issues silently, every time I need to go through all the solved issues to check if they have actually been solved.

Reply:
this one will get fixed by squashing, which i wanted to do as last thing. i will keep it open until that point